operations/sources.md

```diff
@@ -25,6 +34,8 @@ Returns all sources from which reservations can be coming from. Note this operat
 | `ClientToken` | string | required | Token identifying the client application. |
 | `AccessToken` | string | required | Access token of the client application. |
 | `Client` | string | required | Name and version of the client application. |
+| `SourceIds` | array of string | optional, max 1000 items | Unique identifiers of [Source](sources.md#source). |
+| `UpdatedUtc` | [Time interval](#time-interval) | optional, max length 3 months | Interval in which the [Source](sources.md#source) was updated. |

```

Comment:
It's a minor point, but it's nice if the order of the properties in the examples are the same as the order of the properties in the properties table. Here they are the other way around.

Reply:
It's a good point! 🚀